docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,29 +19,29 @@ helpviewer_keywords:
 ---
 # Code-style naming rules
 
-In your `.editorconfig` file, you can define **naming rules** that specify and enforce how .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;should be named. For example, you can specify that public members must be capitalized, or that private fields must begin with `_`.
+In your `.editorconfig` file, you can define naming conventions for your .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;and how the compiler or IDE should enforce those conventions. For example, you could specify that a public member that isn't capitalized should be treated as a compiler error, or that if a private field doesn't begin with an `_` a build warning should be issued.
 
-A naming rule has three components:
+Specifically, you can define a **naming rule**, which consists of three parts:
 
 * The **symbol group** that the rule applies to, for example, public members or private fields.
 * The **naming style** to associate with the rule, for example, that the name must be capitalized or start with an underscore.
-* The severity for enforcing the convention.
-
-First, you must specify the symbol group and naming style and give each of them a title. Then you specify the naming rule, which links everything together.
+* The severity level of the message when code elements included in the symbol group don't follow the naming style.
 
 ## General syntax
 
-To define a naming rule, symbol group, or naming style, set one or more properties using the following syntax:
+To define any of the above entities&mdash;a naming rule, symbol group, or naming style&mdash;set one or more properties using the following syntax:
 
 ```ini
-<kind>.<title>.<propertyName> = <propertyValue>
+<kind>.<entityName>.<propertyName> = <propertyValue>
 ```
 
+Property settings are grouped into entities based on their  shared `kind` and `entityName`.
```

Comment:
```suggestion
Property settings are grouped into entities based on their shared `kind` and `entityName`.
```

Reply:
I'm not sure that this sentence will be easily understood.

Reply:
I'm open to other formulations. The main point I'm trying to get across here is that unlike say C# classes or XML elements, there's no special syntax for defining the parent entity; it's composed only of the various property settings.

Reply:
This is saying the same thing as the previous paragraph:

> To define any of the above entities—a naming rule, symbol group, or naming style—set one or more properties using the following syntax:

This paragraph doesn't add anything more; and it can be removed.

Reply:
Is this better?

> All the property settings for a given `kind` and `entityName` make up that specific entity definition.